### PR TITLE
nuvie 0.5 (new formula)

### PR DESCRIPTION
**Pull request by tomyun**
[Nuvie](http://nuvie.sourceforge.net/) is an open source engine for playing Ultima 6.

Default paths are modified to better conform Homebrew style.
- Game directory: `/usr/local/var/nuvie/game/ultima6/`
- Save directory: `/usr/local/var/nuvie/savegames/`
- Default config: `/usr/local/var/nuvie/nuvie.cfg`
- Default data: `/usr/local/Cellar/nuvie/[version]/share/data/`

No `test` block was made due to limited command line arguments.

### CODE REVIEW

File: nuvie.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      s.gsub! "/Library/Application Support/Nuvie/", "#{var}/nuvie/"
+    end
+    system "./autogen.sh" if build.head?
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-sdltest",
+                          "--prefix=#{prefix}"
+    system "make"
+    bin.install "nuvie"
+    share.install "data"
```
Comment (mistydemeo) [inline nuvie.rb]: Would it be possible to put this in `pkgshare` instead? This'll get linked into `/usr/local/share/data`, which may conflict with other packages.

Comment (tomyun) [inline nuvie.rb]: Oops, didn't know `share` got linked globally. Now changed to `pkgshare`. Thanks!

**Comment by mistydemeo** [comment]
Thanks! Do we need to alter a config file to let it know where the data has been placed?

**Comment by tomyun** [comment]
Yes, according to [the manual](http://nuvie.sourceforge.net/?page=docs). Unlike other game interpreters, Nuvie doesn't seem to support any command line arguments for setting up paths.

File: nuvie.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      #{var}/nuvie/game/ultima6/
+    Save games will be stored in the following directory:
+      #{var}/nuvie/savegames/
+    Config file will be located at:
+      #{var}/nuvie/nuvie.cfg
+    EOS
+  end
+
+  test do
+    pid = spawn "#{bin}/nuvie u6 --end", :err => "log"
```
Comment (mistydemeo) [inline nuvie.rb]: Unfortunately this isn't available in Ruby 1.8.

Comment (tomyun) [inline nuvie.rb]: `test` block is removed. I fiddled with many approaches, but couldn't manage to get it right for 1.8.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**